cs/src/Contracts/TunnelPort.cs

```diff
@@ -99,5 +99,14 @@ public TunnelPort()
         /// </summary>
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
         public TunnelPortStatus? Status { get; set; }
+
+        /// <summary>
+        /// Gets or sets the username for the ssh service user is trying to forward.
+        /// </summary>
+        /// <remarks>
+        /// Should be provided if the <see cref="TunnelProtocol"/> is Ssh.
+        /// </remarks>
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public string? SshUser { get; set; }
```

Comment:
It's not clear to me how this property is used. Is it set by the user/application when adding the port? In what scenario?

Reply:
This is used as part of authentication flow from Tunnels Service to the user's ssh server. To give additional background, user authentication from his `ssh client` to `Tunnels Service` happens with his github public ssh key. And after that authentication, `Tunnels Service` has to authenticate with his `sshd` host. And the current way we are supporting this is asking him to add `Tunnels Service` ssh public key to his `~.ssh/authorized_keys` file and provide the corresponding username as this parameter.